doc/source/examples/extended_examples/executable/cli_rotor.py

```diff
@@ -0,0 +1,111 @@
+# Script to calculate the first natural frequecy of a rotor for a given set of properties
+import click
+
+# Importing packages
```

Comment:
```suggestion
# Import packages
```

Reply:
Yes... because of the include they might appear later too.